test/system/030-run.bats

```diff
@@ -1006,6 +1006,24 @@ EOF
     CONTAINERS_CONF="$containersconf" run_podman 1 run --rm --read-only-tmpfs=false $IMAGE touch /tmp/testro
 }
 
+@test "podman run ulimit from containers.conf" {
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    nofile=$((RANDOM % 10000 + 5))
+    nofile2=$((RANDOM % 10000 + 5))
+    cat >$containersconf <<EOF
+[containers]
+default_ulimits = [
+  "nofile=${nofile}:${nofile}",
+]
+EOF
+
+    CONTAINERS_CONF="$containersconf" run_podman run --rm $IMAGE grep "Max open files" /proc/self/limits
+    assert "$output" =~ "${nofile} * ${nofile}"
```

Comment:
That should be `" ${nofile}  * ${nofile}  * files"` (this is why I suggested copypasting).

Probably not worth repushing for, but please fix if you need to repush for any other reason. Same below.

Reply:
...and, it looks like you need to repush, so please address this too.

Also, could you replace `.Contents()` with `.OutputToString()` to make failures less painful?